tools/release_info/release_info_config_OSS.json

```diff
@@ -56,6 +56,7 @@
    # of this component between two releases of the RobotFramework AIO. Between these scopes there is not necessarily a full match.
    
    "RELEASE_ITEM_FILES" : [
+                            "../../config/robotframework_aio/release_items_Robotframework_AIO.json",

```

Comment:
Because of changes in this file please also adapt minor version number and date (above):

"VERSION_CONFIG" : "0.1.0 / 21.11.2023 (OSS version)",

Reply:
Hi Holger,

Thank you for your finding. I have fixed it in new commit.

Ngoan